src/digital.jl

```diff
@@ -8,8 +8,8 @@ for (cfunction, jfunction, ntask) in (
     end
     @eval function $jfunction(t::$ntask, channel::String)
         catch_error( $cfunction(t.th,
-                Ref(codeunits(channel),1),
-                Ref(codeunits(""),1),
+                Ref(codeunits(channel)),
+                Ref(codeunits("")),
```

Comment:
the second arg to Ref might default to 1 if the first is an array.  but i'm not sure.  should probably put it back in.

Reply:
I fixed this, in testing, the 1 is necessary.